templates/rhdm70-full.yaml

```diff
@@ -0,0 +1,512 @@
+        from:
+          kind: ImageStreamTag
+          namespace: "${IMAGE_STREAM_NAMESPACE}"
+          name: rhdm70-decisioncentral-openshift:1.0
+    - type: ConfigChange
+    replicas: 1
+    selector:
+      deploymentConfig: "${APPLICATION_NAME}-rhdmcentr"
+    template:
+      metadata:
+        name: "${APPLICATION_NAME}-rhdmcentr"
+        labels:
+          deploymentConfig: "${APPLICATION_NAME}-rhdmcentr"
+          application: "${APPLICATION_NAME}"
+      spec:
+        serviceAccountName: decisioncentral-service-account
+        terminationGracePeriodSeconds: 60
+        containers:
+        - name: "${APPLICATION_NAME}-rhdmcentr"
+          image: rhdm70-decisioncentral-openshift
+          imagePullPolicy: Always
+          volumeMounts:
+          - name: decisioncentral-keystore-volume
+            mountPath: "/etc/decisioncentral-secret-volume"
+            readOnly: true
+          - name: "${APPLICATION_NAME}-rhdmcentr-pvol"
+            mountPath: "/opt/eap/standalone/data/bpmsuite"
+          livenessProbe:
+            exec:
+              command:
+              - "/bin/bash"
+              - "-c"
+              - "/opt/eap/bin/livenessProbe.sh"
+          readinessProbe:
+            exec:
+              command:
+              - "/bin/bash"
+              - "-c"
+              - "/opt/eap/bin/readinessProbe.sh"
+          ports:
+          - name: jolokia
+            containerPort: 8778
+            protocol: TCP
+          - name: http
+            containerPort: 8080
+            protocol: TCP
+          - name: https
+            containerPort: 8443
+            protocol: TCP
+          env:
+          - name: KIE_ADMIN_PWD
+            value: "${KIE_ADMIN_PWD}"
+          - name: KIE_ADMIN_USER
+            value: "${KIE_ADMIN_USER}"
+          - name: KIE_MBEANS
+            value: "${KIE_MBEANS}"
+          - name: KIE_SERVER_CONTROLLER_PWD
+            value: "${KIE_SERVER_CONTROLLER_PWD}"
+          - name: KIE_SERVER_CONTROLLER_USER
+            value: "${KIE_SERVER_CONTROLLER_USER}"
+          - name: KIE_SERVER_PWD
+            value: "${KIE_SERVER_PWD}"
+          - name: KIE_SERVER_USER
+            value: "${KIE_SERVER_USER}"
+          - name: HTTPS_KEYSTORE_DIR
+            value: "/etc/decisioncentral-secret-volume"
+          - name: HTTPS_KEYSTORE
+            value: "${DECISION_CENTRAL_HTTPS_KEYSTORE}"
+          - name: HTTPS_NAME
+            value: "${DECISION_CENTRAL_HTTPS_NAME}"
+          - name: HTTPS_PASSWORD
+            value: "${DECISION_CENTRAL_HTTPS_PASSWORD}"
+          - name: ADMIN_USERNAME
+            value: "${ADMIN_USERNAME}"
+          - name: ADMIN_PASSWORD
+            value: "${ADMIN_PASSWORD}"
+          - name: PROBE_IMPL
+            value: probe.eap.jolokia.EapProbe
+          - name: PROBE_DISABLE_BOOT_ERRORS_CHECK
+            value: 'true'
+        volumes:
+        - name: decisioncentral-keystore-volume
+          secret:
+            secretName: "${HTTPS_SECRET}"
```

Comment:
@psiroky Please change this and other occurrences of HTTPS_SECRET to DECISION_CENTRAL_HTTPS_SECRET or KIE_SERVER_HTTPS_SECRET

Reply:
Thanks, will fix that.